src/pyJiraCli/cmd_export.py

```diff
@@ -233,4 +236,129 @@ def _export_ticket_to_file(issue_key:str, filepath:str, user:str, pw:str, csv:bo
             # export file to json format
             ret_status = issue.create_json(filepath)
 
+    server.logout()
+
     return ret_status
+
+
+def _get_filename(file:str, csv:bool) -> str:
```

Comment:
Return type doesn't match.
The function argument "file" sounds like "filename" but with extension. Its not the file itself. Please choose consistent wording over all places.